### ISSUE TITLE
export-pg-from-queries: "Found unexpected parameters: [--exclude-type-definitions, --edge-label-strategy, edgeAndVertexLabels]"

### ISSUE THREAD

**Issue by shoegazerstella** [aws/neptune-export#44]
I am trying to run the `export-pg-from-queries` command.

Here's my config file:

```
#!/bin/bash

if [ -z "$NEPTUNE_ENDPOINT" ]; then
    echo "Must provide NEPTUNE_ENDPOINT in environment"
    exit 1
fi

json=$(cat <<-END
 {  
    "command": "export-pg-from-queries",
    "outputS3Path" : "s3://xxxx/neptune-export/",
    "jobsize": "medium",

    "params": {
      "endpoint" : "${NEPTUNE_ENDPOINT}",
      "profile" : "neptune_ml",
      "useIamAuth" : true,
      "format": "csv",
      "cloneCluster" : false,
      "queriesFile": "./queries.json"
    },

    "additionalParams": {
      "neptune_ml": {
        "version": "v2.0",
        "targets": [
          { "edge": ["xx", "xx", "xx"],
             "type" : "link_prediction"},
          { "edge": ["xx", "xx", "xx"],
             "type" : "link_prediction"}
             ],
        "features": [ {"type": "auto"} ]
    }
  }

    }
END
)

echo "${json}" 

java -jar neptune-export.jar nesvc \
  --root-path /tmp/neptune-export \
  --json "${json}"
```

my `queries.json` file looks like:

```
[
    {
      "name": "task",
      "queries": [
        "g.V().hasLabel('xx', 'xx', 'xx').limit(1000).as('node').project('node', 'properties', 'inEdges', 'outEdges').by(valueMap().with(WithOptions.tokens)).by(__.bothE().as('edge').project('edge', 'properties').by(label()).by(valueMap().with(WithOptions.tokens).fold())).by(__.both().fold())"
      ]
    }
  ]
```

The error log is the following:
```
Args after service init: export-pg-from-queries --endpoint xxxxxxx --profile neptune_ml --use-iam-auth --format csv --queries-file ./queries.json -d /tmp/neptune-export/output --exclude-type-definitions --edge-label-strategy edgeAndVertexLabels --merge-files --export-id 20230707_163607
neptune-export.jar: [buildVersion='1.0.5', buildTime='2023-06-05T20:01:00+0000', commitId='xxxx', commitTime='2023-06-05T20:00:24+0000']
Found unexpected parameters: [--exclude-type-definitions, --edge-label-strategy, edgeAndVertexLabels]
```

It seems those params have some default but this particular command does not admit them. 
How can I fix this?

Thanks a lot in advance for your help!

**Comment by Cole-Greer** [comment]
Hi @shoegazerstella, it appears that the ML v2 profile will inject those 2 additional CLI args for all property graph exports. Those arguments however are currently not supported by the "export-pg-from-queries" command. I have pushed a quick update to this [branch](https://github.com/aws/neptune-export/tree/issue44) with a quick fix which simply removes those arguments from the command. This may be enough to get you unblocked here although I have not yet tested this with any ML workflows. If you want to try it out you can build it with a simple `mvn clean install` and run the jar from `target/neptune-export-1.0.6-SNAPSHOT-all.jar`.

**Comment by iansrobinson** [comment]
export-pg-from-queries isn't supported for the Neptune ML workflow. The process that generates the training configuration file exports node and edge files formatted in a certain way, which export-pg-from-queries can't guarantee (your queries may produce the correctly structured output, but that's not necessarily the case in general).

So while removing those args from the command may help the export complete, I wouldn't expect the output to necessarily be useable by the training configuration step, or by the training itself.

If you don't need the entirety of your graph for the ML job, consider using the different filtering options in export-pg instead: https://docs.aws.amazon.com/neptune/latest/userguide/export-filtering-examples.html

**Comment by Cole-Greer** [comment]
Resolved as of v1.1.3
